src/Symfony/Component/Console/Question/Question.php

```diff
@@ -62,6 +62,18 @@ public function getDefault()
         return $this->default;
     }
 
+    /**
+     * Changes the default answer.
+     *
+     * @return mixed
+     */
+    public function setDefault($default = null)
+    {
+        return $this->default = $default;
+
+        return $this;
```

Comment:
This looks wrong

Reply:
What are you objecting to?  Should we:

- Get rid of the default value for the function (null)
- Return the default instead of $this (I was just copying what the other set...() functions do)
- Get rid of these changes altogether and make $this->default "protected" instead of "private"
- Something else

I can't fix the problems unless I know what they are :).

Reply:
I mean you have 2 `return` statements